packages/cfd/src/Containers/mt5-compare-table-content.tsx

```diff
@@ -123,6 +125,7 @@ const content: TModalContentProps[] = [
                     'Choosing this jurisdiction will give you a Financial STP account. Your trades will go directly to the market and have tighter spreads.'
                 ),
             },
+            derivx: { text: localize('St. Vincent & Grenadines') },
```

Comment:
Why do we need `Deriv X` related info here? 
Is this part of the requirements?

Reply:
for the modal that we should show to the trading-hub we need to show mt5 and derivx details together.
![image](https://user-images.githubusercontent.com/82078941/194994767-306a42b7-bf9b-4bf8-ab19-2646f2e3bd8f.png)